markdown/org/docs/designs/senya/instructions/en.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Sandy circle skirt: Sewing Instructions"
```

Comment:
You forgot to update this title :)

Reply:
Whoops, removed any evidence of my misdeeds.